docs/2_4_common_schema.adoc

```diff
@@ -1428,8 +1427,8 @@ All variables listed in this element must have attribute `derivative` (in order
 
 |`reinit`
 |[[reinit,`reinit`]]
-Only for Model Exchange (if only a Co-Simulation FMU, this attribute must not be present.
-If both Model Exchange and a Co-Simulation FMU, this attribute is ignored for co-simulation): +
+Only for Model Exchange (if only a Co-Simulation or Scheduled Execution FMU, this attribute must not be present.
+If both Model Exchange and a Co-Simulation or Scheduled Execution FMU, this attribute is ignored for Co-Simulation and Scheduled Execution): +
```

Comment:
Should we reomve the requirement to not have it for co-simulation or scheduled execution, isn't it sufficient to, to stat that is it ignored for those cases?

Reply:
I see potential for making erroneous modelDescription.xml say that you can export as both ME and cs but choose only CS, then it is high risk that this gets there any way.

Reply:
Fixed as suggested, now:
Only used in Model Exchange, ignored for the other interface types.